ca_drought/data/main.js.mustache

```diff
@@ -0,0 +1,441 @@
+var map;
+var timesteps = {{{timesteps}}};
```

Comment:
agreed. this is a temporary hack and we really shouldn't need to do it. rendering with this .js works fine, and as long as we don't need to change that code, I would expect it to be fine.